oem/ibm/libpldmresponder/oem_ibm_handler.cpp

```diff
@@ -283,6 +404,16 @@ void pldm::responder::oem_ibm_platform::Handler::buildOEMPDR(
     buildAllCodeUpdateSensorPDR(this, PLDM_OEM_IBM_ENTITY_FIRMWARE_UPDATE,
                                 ENTITY_INSTANCE_0,
                                 PLDM_OEM_IBM_VERIFICATION_STATE, repo);
+
+    buildAllSystemPowerStateEffecterPDR(
+        this, PLDM_OEM_IBM_CHASSIS_POWER_CONTROLLER, ENTITY_INSTANCE_0,
+        PLDM_STATE_SET_SYSTEM_POWER_STATE, repo);
+
+    pldm_entity fwUpEntity = {PLDM_OEM_IBM_ENTITY_FIRMWARE_UPDATE, 0, 1};
+    attachOemEntityToEntityAssociationPDR(
```

Comment:
like this - pldm_entity powerStateEntity = {PLDM_OEM_IBM_CHASSIS_POWER_CONTROLLER, 0,
                                    1};

Reply:
`pldm_entity powerStateEntity = {PLDM_OEM_IBM_CHASSIS_POWER_CONTROLLER, 0,
                                    1};
    attachOemEntityToEntityAssociationPDR(
        this, bmcEntityTree, "/xyz/openbmc_project/inventory/system", repo,
        powerStateEntity);`

Reply:
done